addon/utils/violations-helper.ts

```diff
@@ -81,13 +85,13 @@ export class ViolationsHelper {
    * @public
    * @return {Array}
    */
-  filterBy(key, value) {
-    if (key === 'rule') {
+  filterBy(key: keyof Result, value: any) {
+    if (<any>key === 'rule') {
       key = 'id';
     }
 
     return this.violations.filter((violation) => {
-      if (key === 'node') {
+      if (<any>key === 'node') {
```

Comment:
This (existing code) seems wrong. There is no `node` key on an `axe.Result`, so I'm not really sure what this code was trying to do.